source/SnapDump-Core/SDSnapshot.class.st

```diff
@@ -21,6 +21,18 @@ Class {
 	#category : #'SnapDump-Core'
 }
 
+{ #category : #accessing }
+SDSnapshot class >> apiVersion [
```

Comment:
Shouldn't that be a method in SDServer or SDOpenApiCall? Or at least an extension method from server package?

Reply:
this #apiVersion is also used by SnapDump Handler to "compute" the url to use in order to reach the endpoint.
That's why I looked for a place shared between all images.

Reply:
Good point!